Cython/Utility/TypeConversion.c

```diff
@@ -823,8 +823,36 @@ static CYTHON_INLINE PyObject* {{TO_PY_FUNCTION}}({{TYPE}} value) {
     {
         int one = 1; int little = (int)*(unsigned char *)&one;
         unsigned char *bytes = (unsigned char *)&value;
+#if !CYTHON_COMPILING_IN_LIMITED_API
         return _PyLong_FromByteArray(bytes, sizeof({{TYPE}}),
                                      little, !is_unsigned);
+#else
+        PyObject *from_bytes, *result;
+        PyObject *py_bytes = NULL, *arg_tuple = NULL, *kwds = NULL, *order_str = NULL;
+        from_bytes = PyObject_GetAttrString((PyObject*)&PyInt_Type, "from_bytes");
```

Comment:
Ok, that looks expensive. But having it work at all is probably still better than having it fail..

Reply:
This is the fallback code for "int type is bigger than long long" or "int type is bigger than long and Python doesn't define `PY_LONG_LONG`". So practically I doubt it gets used much.

But yeah, I expect it's expensive.

Reply:
BTW, `PY_LONG_LONG` is now always defined in recent CPython versions. I think since back when they switched to C99 or a bit later, when they noticed that their code actually doesn't compile without `HAVE_LONG_LONG` any more. :)